integ-test/src/test/java/com/amazon/opendistroforelasticsearch/sql/ppl/QueryAnalysisIT.java

```diff
@@ -86,12 +85,6 @@ public void evalCommandShouldPassSemanticCheck() {
   /**
    * Commands that fail semantic analysis should throw {@link SemanticCheckException}.
    */
-  @Test
```

Comment:
why remove this IT? it should fail syntax check, right?

Reply:
yes it will fails syntax since I removed unsupported stats functions from grammar file.

Reply:
Created another PR to add syntax check: https://github.com/opendistro-for-elasticsearch/sql/pull/523